docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+# https://docs.docker.com/samples/rails/
+
+version: "3.9"  # optional since v1.27.0
+services:
+    db:
+        image: postgres:11
+        ports:
+            - "5432:5432"
+        environment:
+            - POSTGRES_USER=postgres
+            - POSTGRES_PASSWORD=password
+            - POSTGRES_DB=jinglejam
+    web:
+        build: .
+        command: bash -c "rm -f /jinglejam/tmp/pids/server.pid && RAILS_ENV=development bundle exec rails server -p 3000 -b 0.0.0.0'"
```

Comment:
I don't know what this command is supposed to do - it duplicates commands you are already running in your `docker-entrypoint.sh` on container start so having it in here seems unnecessary.

Reply:
This is no longer applicable, I couldnt figure out why the command wasnt doing anything. Turns out you need to put exec "$@" at the end of your entrypoint, so I cleaned it up.